datafusion/src/execution/context.rs

```diff
@@ -1289,14 +1245,45 @@ impl TaskContext {
             runtime,
         }
     }
+
+    /// Return the SessionConfig associated with the Task
+    pub fn session_config(&self) -> SessionConfig {
+        let task_settings = &self.task_settings;
```

Comment:
Rename `task_settings` to `properties` as suggested in the previous PR.

Reply:
Sure